app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,17 @@
+class StaticPagesController < ApplicationController
+  def home
+    # home
+  end
```

Comment:
```suggestion
  def home; end
```
self check các action bên dưới nhé